console/src/main/java/org/georchestra/console/dao/AdminLogDao.java

```diff
@@ -38,4 +40,6 @@
 
     List<AdminLogEntry> findByTarget(String target, Pageable range);
 
+    List<AdminLogEntry> myFilter(@Param("targets") Collection<String> targets, Pageable range);
```

Comment:
`myFilter` is not a very explicit method naming

Reply:
Done with 8ae2b5d